at_command.c

```diff
@@ -338,34 +333,42 @@ EXPORT_DEF int at_enque_sms (struct cpvt* cpvt, const char* destination, const c
  * \param code the CUSD code to send
  */
 
-EXPORT_DEF int at_enque_ussd (struct cpvt * cpvt, const char * code, attribute_unused const char * u1, attribute_unused unsigned u2, attribute_unused int u3, void ** id)
+EXPORT_DEF int at_enque_ussd (struct cpvt *cpvt, const char *code)
```

Comment:
I did the inverse, adding back all missing parameters so we don't need to do any dangerous function-pointer casting.

Reply:
Probably we should add a switch/case to the send2 function. Would be the same performance-wise and we wouldn't screw up the API. I can do that.